### PR TITLE
Remove configurations not yet in place

### PR DESCRIPTION
**Pull request by DannyS03**
- localpaloma not yet available to use (containerized version of network)
- sdk pending

### CODE REVIEW

File: docs/src/guide/develop/palomain/initial-setup.md [hunk 1]
```diff
@@ -48,15 +46,13 @@ Use npm to install the palomain command-line tool globally:
 npm install -g @paloma-money/palomain
 ```
 
-## Next steps: localpaloma or Testnet
+## Next steps: Testnet
 
-Depending on your setup, you can either install localpaloma or use the Pisco testnet to power palomain.
-
-localpaloma is a development environment designed to make it easy for smart contract developers to test their contracts locally. palomain can also interact with the Paloma blockchain's live testing environment using the Pisco testnet.
+paloma-testnet-6 is a development environment designed to make it easy for smart contract developers to test their contracts locally. palomain can also interact with the Paloma blockchain's live testing environment using the Pisco testnet.
```
Comment (verabehr) [inline docs/src/guide/develop/palomain/initial-setup.md:51]: same comments as above. Also palomain and pisco don't exist on paloma

File: docs/src/guide/develop/palomain/using-palomain-localpaloma.md [hunk 2]
```diff
@@ -1,11 +1,10 @@
-# Use palomain with localpaloma
+# Use palomain with paloma-testnet-6
 
-localpaloma is a complete Paloma testnet and ecosystem containerized with Docker. Use localpaloma to simulate transactions in a test environment.
+paloma-testnet-6 is a complete Paloma testnet. 
+Use paloma-testnet-6 to simulate transactions in a test environment.
```
Comment (verabehr) [inline docs/src/guide/develop/palomain/using-palomain-localpaloma.md:5]: same comment as above

**Review by verabehr — CHANGES_REQUESTED** [review]
I didn't go through all lines as it seems to be the same issue throughout. Please refer to the first few comments, but it applies to teh whole PR

**Review by verabehr — COMMENTED** [review]
putting this on hold until we re-review and paloma.js is ready for use.

**Comment by DannyS03** [comment]
@verabehr this was the PR that showcases the SDK docs: https://github.com/palomachain/paloma-docs/pull/7, while this one is mainly removing localpaloma

**Pull request merged**